packages/lib-utils/src/components/resource-summary/resource-summary.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { render, screen } from '@testing-library/react';
+import { toHaveNoViolations, axe } from 'jest-axe';
+import React from 'react';
+import '@testing-library/jest-dom';
+import type { K8sResourceCommon } from '../../index';
+import { ResourceSummary } from './index';
+
+expect.extend(toHaveNoViolations);
+
+const k8Object: K8sResourceCommon = {
+  apiVersion: 'v1beta1',
+  apiGroup: 'tenancy.kcp.dev',
+  kind: 'Workspace',
+  metadata: {
+    name: 'demo-workspace',
+    labels: {
+      label1: 'value1',
+      label2: 'value2',
+    },
+    creationTimestamp: '2022-09-15T21:07:32Z',
+    ownerReferences: [
+      {
+        name: 'owner1',
+        apiVersion: 'apps/v1',
+        kind: 'StatefulSet',
+        uid: '8f5ce882-4a20-47da-ad6a-16d0c4846405',
+      },
+      {
+        name: 'owner2',
+        apiVersion: 'apps/v1',
+        kind: 'StatefulSet',
+        uid: '8f5ce882-4a20-47da-ad6a-16d0c4846406',
+      },
+    ],
+  },
+};
+
+describe('ResourceSummary', () => {
+  it('is accessible', async () => {
+    const { container } = render(<ResourceSummary resource={k8Object} />);
+    const results = await axe(container);
```

Comment:
@kdoberst out of curiosity - are you suggesting having separate specs for axe testing, or should we incorporate these assertions into our normal tests? I'm wondering how this will scale when many props/permutations are being tested in larger components going forward.

Reply:
I've seen it done both ways.  I tend to like to have a test(s) whose primary function is to check for accessibility.  It also makes it very evident in the list of tests run that accessibility was checked vs. hidden as an assert.

Reply:
Chatted with @kdoberst offline - maybe in the future we revisit this to fold these into our main test suites, but for now it's nice seeing the explicit accessibility tests.